app/views/students/_form.html.erb

```diff
@@ -0,0 +1,196 @@
+<%= form_for @student , class: "form-horizontal" do |f| %>
+<% if @student.errors.any? %>
+  <div id="error_explanation">
+    <div class="alert alert-danger">
+      <%= pluralize(@student.errors.count, "error") %> prohibited this student from being saved:
+    </div>
+     
+     <%end%>
+ <!--error...............-->
+
+   <div class="form-horizontal">
+    <div class="row">
+
+  <div class="col-md-6">
+   <div class="form-group">
+<%= f.label :enrollment_no,class: "col-sm-4"%>
```

Comment:
We will make this reusable in next meeting. Please remind me for this. Behavior line 14-27 is repeated so many times.